utils/gear-hooks/src/hooks/api/useCalculateGas/index.ts

```diff
@@ -0,0 +1,46 @@
+import { GasInfo, Hex, Metadata } from '@gear-js/api';
+import { useContext } from 'react';
+import { AnyJson } from '@polkadot/types/types';
+import { AccountContext, ApiContext } from 'context';
+import { CalculateGas, HandleOptions, InitOptions } from './types';
+
+function useCalculateGas(
+  programBuffer: Buffer | undefined,
+  metadata?: Metadata | undefined,
+  options?: InitOptions,
+): CalculateGas;
+function useCalculateGas(
+  programIdOrBuffer: Hex | Buffer | undefined,
```

Comment:
Did you mean codeId? If it is used for `createProgram` extrinsic then this should be called codeId, not programId